### ISSUE TITLE
Produce Public Relations Content for the ESIP Community Ontology Repository (COR)

### ISSUE THREAD

**Issue by lewismc** [ESIPFed/gsoc#18]
For some time now we've been needing to make COR easier to use and to improve the information from a user perspective.
This project is aimed at the following
* undertake some research to discover what resources the user community would like to see
* compile that feedback, discuss it with mentors and rank what to implement for the GSoC project

Examples of work artifacts which would come out of this project would be the production of screencasts for the following levels of content
* getting started: uploading resources, browsing resources, etc.
* creating content within COR: creating vocabulary and ontology resources, creating resource mappings
* advanced querying: using the SPARQL endpoint
* advanced functionality using the REST API's: understanding what functionality the REST API provides, generating clients, providing input on additions to the REST API's.

The aim would be for all of this content to live under the ESIP Youtube account and also for this project to conclude with a Webinar to the [ESIP IT&I Committee](http://wiki.esipfed.org/index.php/Interoperability_and_Technology)

**Comment by lewismc** [comment]
I think we could easily partner with the Education Committee on this GSoC project. Maybe we could have someone from that community onboard as a mentor?

**Comment by FabricioLayedra** [comment]
Hello! I think visualization is a good aproach to increase browsing resources in your platform.

**Comment by ritwikagarwal** [comment]
Mailed you @lewismc

**Comment by timannamit5** [comment]
I am interested in working on the issue and can work on the issue.

**Comment by ritwikagarwal** [comment]
Hey @timannamit5 I suggest you to look over the refrenced repositories first i.e Implement advanced features in the Community Ontology Repository (COR) #12 and look over the code base and play with it understand what it is meant for and then look over the description of the project.
Hope you put up a strong proposal .
Ciao

**Comment by timannamit5** [comment]
@lewismc sir, I went through the description. Can you please tell me, where the source code is present? As there is a COR repository and something in GSOC.

**Comment by lewismc** [comment]
@timannamit5 as mentioned in #12 you can find links to the COR as follows
> In the meantime please feel free to start coding some open issues at either of the following 
> https://github.com/mmisw/orr-ont/
> https://github.com/mmisw/orr-portal

You should familiarize yourself deploying a local instance of COR via Docker as follows https://github.com/mmisw/orr-instance-template.git

**Comment by aditya-kamath** [comment]
Hi @lewismc @ritwikagarwal , I am a student who is interested in this project. 
from what I understand, this project aims at getting familiar with the ESIP COR and then creating content and resources to guide users through the repository. This would also mean getting feedback and suggesting implementations. 

I will get started with looking over at the repository and possibly working on some issues as you suggest.

**Comment by lewismc** [comment]
@timannamit5 @aditya-kamath we are thinking of running this project as a [Google Season of Docs](https://developers.google.com/season-of-docs/docs/) project. Would either of you be open to that or would you like to code on the COR project? Thank you

**Comment by aditya-kamath** [comment]
I have had a look at the Season of Docs timeline. I intend to start my PhD in September this year, I am looking for a project for the summer period between May-August. I would be happy to code for the COR project.

**Comment by ritwikagarwal** [comment]
@lewismc Sir just a thought,we could do this thing in a two way step the documentation part in Google Season of Docs and coding part here so we can keep this issue in both the projects alive??
Thanks

**Comment by ritwikagarwal** [comment]
@timannamit5 did you tried  deploying a local instance of COR via Docker because that will give you a fair idea of the project

**Comment by timannamit5** [comment]
> @timannamit5 did you tried deploying a local instance of COR via Docker because that will give you a fair idea of the project

Yes, sir,  I have deployed a local instance of COR. What next should I aim to work at?

**Comment by timannamit5** [comment]
> @timannamit5 @aditya-kamath we are thinking of running this project as a [Google Season of Docs](https://developers.google.com/season-of-docs/docs/) project. Would either of you be open to that or would you like to code on the COR project? Thank you

Sir, I would like to work for Google Season for Docs. I am open for both the work, but if needed I can do the Google Season for Docs project.

**Comment by lewismc** [comment]
@aditya-kamath Ok thanks for letting us know

**Comment by lewismc** [comment]
@ritwikagarwal 
> Sir just a thought,we could do this thing in a two way step the documentation part in Google Season of Docs and coding part here so we can keep this issue in both the projects alive??
Thanks

Not much coding at all is involved in this project. It is all about documentation.

**Comment by lewismc** [comment]
@timannamit5 as explained various times, there are loads of open issues in the various issues trackers
https://github.com/mmisw/orr-ont/issues
https://github.com/mmisw/orr-portal/issues
Depending on what you are interested in please comment on the issue you would like to start working on.

**Comment by ritwikagarwal** [comment]
@timannamit5 @aditya-kamath gsoc proposal application has opened ! You can start working on the proposals and it will be better if you submit them early for review.

**Comment by esip-lab** [comment]
Hi all - a friendly reminder that there is ONE WEEK LEFT to submit your proposals for this project! Best of luck and we're excited to see what is submitted!

**Comment by lewismc** [comment]
Hi @timannamit5, @aditya-kamath no-one has shared a proposal here or with me directly yet. The final date for this is tomorrow. If you would like to be considered then you MUST submit a proposal by tomorrow this is mandated by Google. Thank you

**Comment by timannamit5** [comment]
Okay, sir. Thank you

**Comment by ritwikagarwal** [comment]
Hey @timannamit5 I reviewed your proposal on Gsoc website !Will get back to you in some minutes however it is quite late for me to review and then propose changes and then submit it back as final still I will try. I have suggested some major changes as the draft seems to be deviating from the original issues .Hope you will look at them.
@aditya-kamath are you planning to submit?
Thanks

**Comment by ritwikagarwal** [comment]
@timannamit5 nice work !Will look forward to read this more briefly.Thanks

**Comment by timannamit5** [comment]
thank you sir
